src/actions/validate.ts

```diff
@@ -39,7 +40,10 @@ async function run(): Promise<void> {
 
     // Validate each commit against Conventional Commit standard
     const messages = await getMessagesToValidate();
-    await validateMessages(messages, config);
+    const convMessges = await validateMessages(messages, config);
```

Comment:
```suggestion
    const convMessages = await validateMessages(messages, config);
```

Though I think the name with prefix of `conv` is a not so clear one.

Reply:
Renamed variables to `commitMessages` and `compliantMessage` to be more concise on their usage.